code/game/machinery/computer/shuttle.dm

```diff
@@ -8,6 +8,9 @@
 
 /obj/machinery/computer/emergency_shuttle/attackby(var/obj/item/weapon/card/W as obj, var/mob/user as mob)
 	if(stat & (BROKEN|NOPOWER))	return
+	if(emergency_shuttle.location != DOCKED)
+		user << "The shuttle is already in motion."

```

Comment:
added this here so it gives people a message when they try to scan their IDs after the shuttle has launched